vscode/package.json

```diff
@@ -38,11 +85,55 @@
         "scopeName": "source.qsharp",
         "path": "./syntaxes/qsharp.tmLanguage.json"
       }
+    ],
+    "debuggers": [
+      {
+        "type": "qsharp",
+        "languages": [
+          "qsharp"
+        ],
+        "label": "Q# Debug",
+        "configurationAttributes": {
+          "launch": {
+            "required": [
+              "program"
+            ],
+            "properties": {
+              "program": {
+                "type": "string",
+                "description": "Absolute path to a text file.",
+                "default": "${file}"
+              },
+              "stopOnEntry": {
+                "type": "boolean",
+                "description": "Automatically stop after launch.",
+                "default": true
+              },
+              "entry": {
+                "type": "string",
+                "description": "Entry expression.",
+                "default": ""
+              },
+              "shots": {
+                "type": "number",
+                "description": "Number of shots to execute.",
+                "default": 10
```

Comment:
If you're debugging, I'd assume 1 would be the better default.

Reply:
When debugging this gets set to `1` in the command implementation for `debugEditorContents`

Reply:
So then what value does setting it to 10 here have?

Reply:
I'll remove it in the next revision. I'd hoped that it would have some effect, but we may be missing how the defaults work.